src/SeqCli/Cli/Commands/License/ShowCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Seq.Api.Model;
+using Seq.Api.Model.License;
+using SeqCli.Cli.Features;
+using SeqCli.Config;
+using SeqCli.Connection;
+using SeqCli.Util;
+using Serilog;
+
+// ReSharper disable once UnusedType.Global
+
+namespace SeqCli.Cli.Commands.License;
+
+[Command("license", "show", "Shows license applied to the Seq server",
+    Example = "seqcli license show")]
+class ShowCommand : Command
+{
+    readonly SeqConnectionFactory _connectionFactory;
+    readonly ConnectionFeature _connection;
+    readonly OutputFormatFeature _output;
+
+    public ShowCommand(SeqConnectionFactory connectionFactory, SeqCliConfig config)
+    {
+        _connectionFactory = connectionFactory ?? throw new ArgumentNullException(nameof(connectionFactory));
+        _connection = Enable<ConnectionFeature>();
+        _output = Enable(new OutputFormatFeature(config.Output));
+    }
+
+    protected override async Task<int> Run()
+    {
+        var connection = _connectionFactory.Connect(_connection);
+        var license = await connection.Licenses.FindCurrentAsync();
+
+        if (license == null)
+        {
+            Log.Warning("No license is currently applied to the server.");
+            return 2;
+        }
```

Comment:
Yes, I think we should remove this. If there's no license applied to the server, since the output format is effectively just what's displayed in the "License text" control, returning `0` here with no output is fine. (The rest of `seqcli` follows a fairly similar set of conventions, e.g. `seqcli config get -k ...` will produce no output if a setting is blank. This makes use in shell scripts easier.)